src/main/java/com/akadk/entity/AbstractEntity.java

```diff
@@ -0,0 +1,28 @@
+package com.akadk.entity;
+
+
+import lombok.Getter;
+import org.springframework.data.annotation.CreatedDate;
+import org.springframework.data.annotation.LastModifiedDate;
+
+import javax.persistence.*;
+import java.util.Date;
+
+@MappedSuperclass
+@Getter
+public abstract class AbstractEntity<T> {
+
+    public abstract T getId();
+
+
+    @Temporal(TemporalType.TIMESTAMP)
+    @Column(name = "created_at", nullable = false, updatable = false, columnDefinition = "TIMESTAMP DEFAULT CURRENT_TIMESTAMP")
+    @CreatedDate
+    private Date createdAt;
+
+    @Temporal(TemporalType.TIMESTAMP)
+    @Column(name = "updated_at", nullable = false, columnDefinition = "TIMESTAMP DEFAULT CURRENT_TIMESTAMP")
+    @LastModifiedDate
+    private Date updatedAt;
+
```

Comment:
If you want them to paste automatically, you should add  @PrePersist and @PreUpdate annotations with initialization functions.